### ISSUE TITLE
Office 2007 - pages move, not windows

### ISSUE THREAD

**Issue by ttencate** [ttencate/taekwindow#37]
Converted from [SourceForge issue 2000149](http://sourceforge.net/support/tracker.php?aid=2000149), submitted by thezero0

Whenr moving a Microsoft Office 2007 window, the page clicked on is moving instead of the full window. This clearly isn't meant to be the case.

Note: Using 0.2.3 (not selectable)

**Comment by ttencate** [comment]
Submitted by thezero0

Fixed in the current development version, I hope:
- Word does use an MDI under the hood, but just doesn't show it. I made Taekwindow pretend that the child windows are not movable/resizable.
- Excel has an MDI, and shows it, but it has its own handling of maximized/unmaximized windows. Hence Taekwindow cannot detect whether an Excel document window is maximized or not. I made it pretend that document windows are not movable/resizable, so you'll always move/resize the main Excel window.
- PowerPoint, Outlook, Publisher and Access do not seem to have any Taekwindow-related issues.
